src/map.h

```diff
@@ -167,14 +167,14 @@ class Map /*: public NodeContainer*/
 	*/
 	virtual MapSector * emergeSector(v2s16 p){ return NULL; }
 
-	// Returns InvalidPositionException if not found
-	MapBlock * getBlockNoCreate(v3s16 p);
 	// Returns NULL if not found
-	MapBlock * getBlockNoCreateNoEx(v3s16 p);
+	MapBlock * getBlockNoCreate(v3s16 p);
 
 	/* Server overrides */
 	virtual MapBlock * emergeBlock(v3s16 p, bool create_blank=true)
```

Comment:
```suggestion
	virtual MapBlock *emergeBlock(v3s16 p, bool create_blank = true)
```

Again, untouched code, but would be nice if fixed :)